.github/workflows/windows.yml

```diff
@@ -714,6 +666,137 @@ jobs:
           path: ${{ env.INSTALL_TEST_DIR }}/TEST*.xml
           if-no-files-found: 'error'
 
+  Python_TensorFlow_Layer_Tests:
```

Comment:
maybe we can keep it as reusable workflow as well? for better granularity

Reply:
will try, I did it like it was already implemented